animations/spells/1st/bless.json

```diff
@@ -0,0 +1,64 @@
+{
+	"effect:aura-bless": [
```

Comment:
The animation will not track with the size of the aura, I rather not animate it. See #2.

Reply:
Makes sense. I will remove the aura section and just have the animation of when they get the buff.